components/CandidateList/CandidateList.tsx

```diff
@@ -1,28 +1,19 @@
-﻿import React from "react";
+﻿import React, {useEffect, useState} from "react";
 import scss from './CandidateList.module.scss';
-import {getCandidates} from "../../api/candidatesApiClient";
+import {Candidate, getCandidates, ListResponse} from "../../api/candidatesApiClient";
 import {CandidateLink} from "../CandidateLink/CandidateLink";
 
+interface CandidatesListProps {
+    fetchCandidates: ListResponse<Candidate>;
```

Comment:
we should probably update the name of this property now :)

Reply:
Have changed it to candidatesList now